Rakefile

```diff
@@ -1,17 +1,10 @@
-require "rake/testtask"
 require "bundler/gem_tasks"
+require "tldr/rake"
 require_relative "lib/tinygql/version"
 
-Rake::TestTask.new(:test) do |t|
-  t.libs << "test"
-  t.warning = true
-  t.verbose = true
-  t.test_files = FileList["test/**/*_test.rb"]
-end
-
 FILES = [ "lib/tinygql/nodes.rb", "lib/tinygql/visitors.rb" ]
 
-task :test => FILES
+task :test => [*FILES, :tldr]
```

Comment:
I don't think this is right.  I think it should just be `task :tldr => FILES`

Reply:
That's fine. I just didn't want to go in and change the testing command `bundle exec rake test` in CI to `tldr` without your OK